src/BasisTheory.net.Test/Model/TokenExtrasTests.cs

```diff
@@ -0,0 +1,70 @@
+/*
+ * Basis Theory API
+ *
+ * ## Getting Started * Sign-in to [Basis Theory](https://basistheory.com) and go to [Applications](https://portal.basistheory.com/applications) * Create a Basis Theory Private Application * All permissions should be selected * Paste the API Key into the `BT-API-KEY` variable
+ *
+ * The version of the OpenAPI document: v1
+ * Generated by: https://github.com/openapitools/openapi-generator.git
+ */
+
+
+using Xunit;
+
+using System;
+using System.Linq;
+using System.IO;
+using System.Collections.Generic;
+using BasisTheory.net.Api;
+using BasisTheory.net.Model;
+using BasisTheory.net.Client;
+using System.Reflection;
+using Newtonsoft.Json;
+
+namespace BasisTheory.net.Test.Model
+{
+    /// <summary>
+    ///  Class for testing TokenExtras
+    /// </summary>
+    /// <remarks>
+    /// This file is automatically generated by OpenAPI Generator (https://openapi-generator.tech).
+    /// Please update the test case below to test the model.
+    /// </remarks>
+    public class TokenExtrasTests : IDisposable
+    {
+        // TODO uncomment below to declare an instance variable for TokenExtras
+        //private TokenExtras instance;
+
+        public TokenExtrasTests()
+        {
+            // TODO uncomment below to create an instance of TokenExtras
+            //instance = new TokenExtras();
+        }
+
+        public void Dispose()
+        {
+            // Cleanup when everything is done.
+        }
+
+        /// <summary>
+        /// Test an instance of TokenExtras
+        /// </summary>
+        [Fact]
+        public void TokenExtrasInstanceTest()
+        {
+            // TODO uncomment below to test "IsType" TokenExtras
+            //Assert.IsType<TokenExtras>(instance);
+        }
+
+
+        /// <summary>
+        /// Test the property 'Deduplicated'
+        /// </summary>
+        [Fact]
+        public void DeduplicatedTest()
+        {
+            // TODO unit test for the property 'Deduplicated'
+        }
+
+    }
+
+}
```

Comment:
@lcschy any idea what this is?

Reply:
its something we are working on around token intents and stuff, adding additional info outside of the token schema for certain types of tokens